src/main/kotlin/com/hedvig/accountservice/schedulers/service/ScheduleChargesServiceImpl.kt

```diff
@@ -0,0 +1,49 @@
+package com.hedvig.accountservice.schedulers.service
+
+import com.hedvig.accountservice.integration.paymentService.PaymentService
+import com.hedvig.accountservice.schedulers.domain.commands.ScheduleChargeCommand
+import com.hedvig.accountservice.schedulers.model.ChargeStatus
+import com.hedvig.accountservice.schedulers.model.SchedulerStateManager
+import com.hedvig.accountservice.schedulers.model.SchedulerStateTransition
+import com.hedvig.accountservice.schedulers.model.SchedulerStateTransitionOutcome
+import com.hedvig.accountservice.schedulers.web.dto.PaymentProvider
+import org.axonframework.commandhandling.gateway.CommandGateway
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class ScheduleChargesServiceImpl(
+    private val commandGateway: CommandGateway,
+    private val stateManager: SchedulerStateManager,
+    private val paymentService: PaymentService
+): ScheduleChargesService {
+    override fun scheduleChargesForAll(scheduledBy: String) =
+        scheduleCharges(stateManager.findAllMemberIds(ChargeStatus.APPROVED_FOR_CHARGE), scheduledBy)
```

Comment:
Some boyscouting maybe

```suggestion
        scheduleCharges(stateManager.findAllMemberIdsByStatus(ChargeStatus.APPROVED_FOR_CHARGE), scheduledBy)
```